third_party/gtp_ovs/ovs-gtp-patches/2.15/dev.sh

```diff
@@ -0,0 +1,78 @@
+#! /bin/bash
+
+set -x
+OVS_VER='2.15'
+DIR="/root/ovs-build"
+
+MAGMA_ROOT="/home/vagrant/magma"
+
+function setup() {
+
+  set -e
+  apt install -y build-essential linux-headers-generic
+  apt install -y dh-make debhelper dh-python devscripts python3-dev
+  apt install -y graphviz libssl-dev python3-all python3-sphinx libunbound-dev libunwind-dev
+  apt install -y linux-modules-extra-`uname -r`
+  pip3 install scapy
+
+  rm -rf $DIR
+  mkdir $DIR
+  cd $DIR
+
+  git clone  https://github.com/openvswitch/ovs
+  cd ovs/
+  git checkout 31288dc725be6bc8eaa4e8641ee28895c9d0fd7a
```

Comment:
Using the `--branch` option for git on clone, we might be able to save to separate checkout action.